examples/oapp-solana/programs/my_oapp/src/errors.rs

```diff
@@ -0,0 +1,6 @@
+use anchor_lang::prelude::error_code;
+
+#[error_code]
+pub enum CounterError {
+    InvalidMessageType
+}
```

Comment:
Consider renaming `CounterError` to a name that reflects its purpose in this messaging OApp implementation (e.g., `OAppError`, `MessageError`).